tests/test_optim.py

```diff
@@ -93,7 +93,7 @@ def test_minimize(tensorlib, precision, optimizer, do_grad, do_stitch):
             'no_grad-minuit-jax-64b': [0.5000493563528641, 1.0000043833614634],
             # do grad, minuit, 32b
             'do_grad-minuit-pytorch-32b': [0.5017611384391785, 0.9997190237045288],
-            'do_grad-minuit-tensorflow-32b': [0.501288652420044, 1.0000219345092773],
+            'do_grad-minuit-tensorflow-32b': [0.5012885928153992, 1.0000673532485962],
```

Comment:
Do we know why this is happening? Seems strange that this is affecting anything. :?

Reply:
it's so tiny. I don't know. But this is just minuit being minuit 🤷‍♂️. It most likely stops optimizing at some point... and for 32b, minuit is pretty junk about.

Reply:
unsatisfying, but fair.